package/com.unity.formats.usd/Runtime/Scripts/Behaviors/UsdAsset.cs

```diff
@@ -590,14 +590,22 @@ public void ExportOverrides(Scene sceneInWhichToStoreTransforms)
             var sceneToReference = this;
             var overs = sceneInWhichToStoreTransforms;
 
+
             if (overs == null)
             {
+                UsdEditorAnalytics.SendExportEvent(Path.GetExtension(sceneToReference.usdFullPath), .0f, false, onlyOverrides: true);
                 return;
             }
 
+            bool success = false;
+
+            System.Diagnostics.Stopwatch analyticsTimer = new System.Diagnostics.Stopwatch();
```

Comment:
Would be good to standardise on either using System.Diagnostics or System.Diagnostics.Stopwatch

same with Unity.Formats.USD.UsdEditorAnalytics.ImportResult

Reply:
What do you mean by this? Do you mean specifically in the 'using's?

Reply:
Yeah pretty much, change looks good thanks!